### ISSUE TITLE
clean build of ceylon-ide-common results in errors in com.redhat.ceylon.eclipse.ui

### ISSUE THREAD

**Issue by gavinking** [ceylon/ceylon-ide-eclipse#1716]
When ever I rebuild the ceylon-ide-common project, I end up with errors in the com.redhat.ceylon.eclipse.ui project, and I have to manually rebuild com.redhat.ceylon.eclipse.ui. @davidfestal do you know why this is?

**Comment by bjansen** [comment]
errors in `modelJ2C.java`?

**Comment by davidfestal** [comment]
yeah, errors on `JDTModule` etc always come back in this file only, and I never found out why.
Just saving this file again and so triggering an (very small) incremental Java build is sufficient to get rid of these errors.

**Comment by gavinking** [comment]
OK.
